app/views/v1/subdomain_util.py

```diff
@@ -0,0 +1,109 @@
+from typing import Set, Tuple
+
+import json
+
+import flask_jwt_extended
+from flask import request, url_for
+
+import app.utils.authentication_utils as authentication_utils
+import app.actions as actions
+import app.db_models as db_models
+
+from . import bp
+from app.utils.ct_search import get_subdomains_from_ct
+from app.actions.add_targets import add_targets
+
+from time import time
+
+BATCH_SIZE = 10  # set appropriate amount
+RESCAN_INTERVAL = 62400  # to check target once a day
```

Comment:
```suggestion
RESCAN_INTERVAL = 24*60*60  # to check target once a day
```

Also, maybe move this to `config.py`.